import-automation/progress-dashboard-rest/app/resource/progress_log.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Import log resources associated with the endpoints
+'/import_attempts/<string:attempt_id>/logs',
+'/system_runs/<string:run_id>/logs', and '/logs/<string:log_id>'.
+"""
+
+import enum
+
+import flask_restful
+from flask_restful import reqparse
+
+from app import utils
+from app.service import validation
+from app.service import import_attempt_database
+from app.service import system_run_database
+from app.service import progress_log_database
+from app.model import import_attempt_model
+from app.model import system_run_model
+from app.model import progress_log_model
+
+_ATTEMPT = import_attempt_model.ImportAttemptModel
```

Comment:
Given that we use these base classes as you wrote, we could name the files and classes more simply so they do not stutter as much, e.g., import_attempt.Model.  It might be easier to rename in a separate code change.

Reply:
Added a TODO